server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -245,17 +245,6 @@ private void killUnusedSegments(
     stats.add(Stats.Kill.SUBMITTED_TASKS, submittedTasks);
   }
 
-  /**
-   * <p>
-   * Calculates the interval for which segments are to be killed in a datasource.
```

Comment:
Why is this javadoc not valid anymore?

Reply:
Please see https://github.com/apache/druid/pull/16094. I didn't see a point in just keeping "Calculates the interval for which segments are to be killed in a datasource." as the method name is quite evident.